### ISSUE TITLE
It should be possible to customize the language, copyright and company name in the exe file meta data

### ISSUE THREAD

**Issue by rekire** [flutter/flutter#68284]
## Use case
I want to customize the language, copyright and company name in the exe file meta data.

The company name is currently limited to the package name which doesn't really make sense.  
The copyright is currently defined as `Copyright (C) {{year}} {{organization}}. All rights reserved.` which makes sense as default as long the `organization` is not the package name.  
I mostly don't write en-US applications so I would prefer to have something else or might be better language independent. You can use that constant too.

## Proposal
The fields from above should be changeable might be in the `pubspec.yaml` the current template file is here:
https://github.com/flutter/flutter/blob/50fd10fc6f09797ff0eb538fe00b6af331bc3100/packages/flutter_tools/templates/app/windows.tmpl/runner/Runner.rc.tmpl

**Comment by gspencergoog** [comment]
Yes, we need a new flag for `flutter create` that specifies this information for the template, and the template would need to be updated.

**Comment by gspencergoog** [comment]
This is really just the template creation phase, and the workaround is to just edit the created file manually.

**Comment by matanlurey** [comment]
It's a non-goal to allow customizing every element of every template from the command-line.

The changes will need to be made manually, or folks can make their own example (starter) repositories filled-in and clone/copy from those.
